pom.xml

```diff
@@ -25,7 +25,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <maven-compiler-plugin.version>3.1</maven-compiler-plugin.version>
         <groovy.version>2.5.7</groovy.version>
-        <httpclient.version>4.3.6</httpclient.version>
+        <httpclient.version>4.4</httpclient.version>
```

Comment:
why do not upgrade to version 4.5.x as dependabot suggests? https://github.com/TouK/http-mock-server/pull/9

Reply:
Right, I upgraded to the version that caused error.